src/components/footer/index.tsx

```diff
@@ -126,22 +142,9 @@ function Footer({
 
 					return (
 						// eslint-disable-next-line react/no-array-index-key
-						<li className={s.linkListItem} key={index}>
-							{innerElement}
-						</li>
+						<li key={index}>{innerElement}</li>
 					)
 				})}
-				<li>
-					<ButtonLink
```

Comment:
Since the switcher stays in the same place between mobile and desktop now we can remove this duplicate markup and the visual order matches the focus order always.